support-frontend/assets/pages/new-contributions-landing/contributionsLandingInit.js

```diff
@@ -115,7 +117,10 @@ function initialisePaymentMethods(state: State, dispatch: Function) {
     });
   }
 
-  if (getQueryParameter('paypal-js') !== 'no') {
+  const hasRecurring = contributionTypes.includes('MONTHLY')
```

Comment:
I wonder if this could be more clearly named, to me it's not immediately clear what exactly has recurring. Maybe `recurringContributionsAvailable` or `recurringContributionsDisplayed`?